calico-cloud/image-assurance/scanners/cluster-scanner.mdx

```diff
@@ -0,0 +1,61 @@
+---
+description: Identify vulnerabilities in Kubernetes clusters.
+---
+
+# Automatically scan using cluster Scanner
```

Comment:
For search results, a better title is: Automatically scan images using cluster scanner

Reply:
fixed with bed5e89